src/platforms/hosted/ftdi_bmp.c

```diff
@@ -382,7 +382,7 @@ int ftdi_bmp_init(bmda_cli_options_s *cl_opts, bmp_info_s *info)
 	}
 
 	if (!cable->name) {
-		DEBUG_WARN("No cable matching found for %s\n", cl_opts->opt_cable);
```

Comment:
This is an old school naming of JTAG debuggers. In commercial software they are often called cables. Not sure how common the `adaptor` naming is for the same thing.

Reply:
So, the motivation for changing this (and we actually started this process 2 releases ago) is that in context of BMDA, the "cable" nomenclature results in an incredibly confusing user experience for anyone not already deeply rooted in the FTDI ecosystem (we might be able to get away with this if we said "FTDI Cable" maybe but it's a stretch), and makes the BMDA code more spaghetti and incomprehensible.

Consider, for example, someone who's just got a Tigard and they don't know FTDI's nomenclature at all - they're going to call the board an adaptor (it is, to be fair), look for options to set what adaptor they're using, and expect error messages consistent with that.

Likewise, within BMDA, "cable" winds up resulting in a reading experience that, from our own past experience, results in head scratching and getting very confused across against the other backends supported.

Hopefully that helps explain the motivation for this trajectory.